Gemfile

```diff
@@ -55,6 +55,7 @@ gem 'gaffe'
 gem 'ruby-filemagic'
 gem 'mime-types'
 gem 'midi-smtp-server'
+gem "table_print"
```

Comment:
adding `table_print` is IMO completely unrelated with the other changes. please create a separate PR and add it to the [`development` group in the Gemfile](https://github.com/foodcoops/foodsoft/blob/1e883f05c0c59641af9f9da2d1b039f752dc1f21/Gemfile#L74), since it's not required for a production deployment

Reply:
Done. Extracted to #935.